spec/components/parameters/endDate.ts

```diff
@@ -0,0 +1,15 @@
+import { ParameterObject } from 'openapi3-ts';
+import { createComponentRef } from '../../../utils/ref';
+
+const endDate: ParameterObject = {
+  name: 'endDate',
```

Comment:
In order to respect the pattern used in the previous APIs, in my opinion, would be better to call this parameter 'endAt'.